src/functions.rs

```diff
@@ -45,6 +54,7 @@ pub mod consts {
 ///
 /// fn main() -> io::Result<()> {
 ///     println!("{}", is_removable("src/lib.rs")?);
+///     #[cfg(unix)]
 ///     println!("{}", is_removable("/root")?);
```

Comment:
Can you remove this line? I think the other examples are enough for this one.

Reply:
Done.